cmd/earthly/main.go

```diff
@@ -627,6 +632,20 @@ func newEarthlyApp(ctx context.Context, console conslogging.ConsoleLogger) *eart
 			Value:       defaultEnvFile,
 			Destination: &app.envFile,
 		},
+		&cli.StringFlag{
+			Name:        "secrets-file",
+			EnvVars:     []string{"EARTHLY_SECRETS_FILE"},
+			Usage:       "Use secrets from this file",
+			Value:       defaultSecretsFile,
+			Destination: &app.secretsFile,
+		},
+		&cli.StringFlag{
+			Name:        "args-file",
+			EnvVars:     []string{"EARTHLY_ARGS_FILE"},
+			Usage:       "Use build args from this file",
+			Value:       defaultArgsFile,
+			Destination: &app.argsFile,
+		},
```

Comment:
Should these be hidden & experimental until Earthly v0.7?

Reply:
Possibly? it might make sense since introducing the warning in a patch version does seem a bit odd. I'll hide them (and the warning too).